public/static/docs/command-reference/checkout.md

```diff
@@ -222,9 +211,9 @@ running `dvc checkout` when needed.
 We can then checkout the master branch again:
 
 ```dvc
-$ git checkout bigrams
-Previous HEAD position was d171a12 add evaluation stage
-HEAD is now at d092b42 try using bigrams
+$ git checkout 9-bigrams-model  # Bigrams version of the model
```

Comment:
so, from what I see `example-get-started` and get-started are broken. It was made to have two tags to define experiment checkpoints (besides tags that define steps). Parts of the get started (other tutorials?) are written with an assumption to have those human readable tags - baseline, bigrams, etc. Looks like does not have them anymore?

Reply:
Yep, they were removed a long time ago. Now it has like 12 numbered tags and the old ones correspond to 9 and 10 or something like that. I don't know if it's "broken" though, I think we just need to update these references.

Reply:
so, it means that get started is broken I think. Also, not sure that was a good idea to remove them - they were made with a certain purpose to use them when you need to focus attention on checkpoints (not steps), also eventually to filter them with `dvc metrics show -T -include="*-experiments"` or something similar.

Reply:
> it means that get started is broken I think

- [ ] I can check.

> sure that was a good idea to remove them...

I didn't do that 😋 -> https://github.com/iterative/example-repos-dev/commit/6fa622d89c714dcccaa60d707d9e190965176918

Please lmk how to fix the situation and priority and I'll be glad to make the updates.

Reply:
yep, so in that link I do see them - tags for experiments I mean. Or I'm not sure I got the point  🤔

Reply:
I was just clarifying I didn't make that change, in case you thought so. But it doesn't matter, please lmk what to do about it and I'll address it.

Reply:
Extracted to #961 – please follow up there if you need to add more info. Thanks

Reply:
ah, okay, sorry .. I see they are still here https://github.com/iterative/example-get-started/releases?after=1-initialize. Then I just don't see any reason to update docs to change names - let's just use those two tags that were made to designate two experiment checkpoints?

Reply:
@jorgeorpinel see my comment ^ . I was wrong, let's just keep using human readable tags where it is more appropriate.

Reply:
My bad. I thought they were gone because they don't come up in page 1 of https://github.com/iterative/example-get-started/tags. Will revert...

Reply:
OK, updated to the actual tag names. They did change from the original version here actually. See fc832079.

>  I can check.

I did and it works well.